sync_data/sync_data/test_settings.py

```diff
@@ -0,0 +1,7 @@
+from .base_settings import *
+
+ACTBLUE_TO_KNACK_MAPPING = {
+    'contribution#createdAt': '888',
+    'lineitems#0#amount': '889',
+    'contribution#contributionForm': '893'
+}
```

Comment:
Hmm, I still feel like this mapping is business logic, not settings, and should be in `views.py`.  Not super fussed about it if you feel strongly otherwise though.

Reply:
I'd rather not have to have both versions in the file - this is a convenient way to split out the test and production mappings